### ISSUE TITLE
TextLabel not childOf CF?

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/ontology#371]
I think TextLabel is potentially going to be used more broadly than for Cultural Formation terms, and the definition isn't specific to CF, so I propose that we remove the ChildOf relationship.

**Comment by joelacummings** [comment]
Agreed, for now I'd say it actually sits under owl:Thing but that will likely change once we select an Upper Level Ontology.
